x-pack/plugins/search_indices/common/doc_links.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DocLinks } from '@kbn/doc-links';
+
+class SearchIndicesDocLinks {
+  public apiReference: string = '';
+
+  constructor() {}
+
+  setDocLinks(newDocLinks: DocLinks) {
+    this.apiReference = newDocLinks.apiReference;
```

Comment:
@joemcelroy Will we need to worry about showing different links here based on serverless and stack?

Reply:
We will do eventually. ATM we rely on stack URLs, waiting on @leemthompo guidance on how that will work out :)

Reply:
@lcawl knows more about this topic than me, and I know the autogenerated API refs are coming along nicely :)

Reply:
We do have the [isServerless](https://github.com/elastic/kibana/blob/main/packages/kbn-doc-links/src/get_doc_links.ts#L47) option too IIUC

Reply:
Yes, the doc link service can have two "flavours" of links so that's definitely something that already exists.

Reply:
@lcawl would you recommend linking to the autogenerated API refs today, or circling back in a while on that one?